open-apparel-registry/open-apparel-registry
# Add architecture diagrams

## Overview

Add architecture diagrams

## Testing Instructions

* Verify the diagrams for correctness and completeness
* Verify the new README

## Checklist
